cluster/tenants.go

```diff
@@ -120,6 +121,20 @@ func TenantAddAction(ctx *Context, name, shortName, userName, userEmail string)
 	}
 	// if the first admin name and email was provided send them an invitation
 	if userName != "" && userEmail != "" {
+		// wait for MinIO to be ready before creating the first user
+		for {
```

Comment:
can we allow a number of trial times or so?
we should not allow the possibility of infinite loops since it could possibly never be ready.

Reply:
implmented